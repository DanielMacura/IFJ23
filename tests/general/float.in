var x = 1.0154e-1
write(x)