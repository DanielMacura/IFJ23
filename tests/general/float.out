0x1.9fe86833c6003p-4